Makefile

```diff
@@ -156,6 +156,6 @@ minikube-start:
 minikube-delete:
 	@./scripts/minikube-delete.sh
 
-install: build
-	EPINIO_DONT_WAIT_FOR_DEPLOYMENT=1 ./dist/epinio-linux-amd64 install --skip-default-namespace
+install: embed_files build-$(shell uname | tr "[:upper:]" "[:lower:]")-amd64
```

Comment:
I wonder if that name is too generic? It's installing for CI, not useful for users. More like a CI/dev Makefile target?

Also, when this builds the platform binary, do tests still need to build it again (https://github.com/epinio/epinio/blob/main/acceptance/testenv/epinio.go#L125-L136), or is the server binary enough?
Hm, probably doesn't matter, as rebuilding is fast.

Reply:
Removed BuildEpinio from tests and also removed generic bash call.

Reply:
Maybe `install` :arrow_right:  `ci_setup` ?

Reply:
Done